commands/rod.txt

```diff
@@ -6,7 +6,7 @@
     "fields": [
       {
         "name": "__General Information & Benefits__",
-        "value": "*Note: more details available [here](https://runescape.wiki/w/Ring_of_Death)*\n⬥ **Effect**: revived to full HP, 3 min cooldown; activates before Sign of Life <:signoflife:1005552211847295076>\n\u00a0\u00a0\u00a0\u00a0• A ramping DoT is applied for 3 minutes.\n\u00a0\u00a0\u00a0\u00a0\u00a0\u00a0\u00a0\u00a0- Taking off <:RoD:513190159462825984> triples the damage until it is re-equipped\n⬥ Effect is removed on many phase transitions → can be used near-losslessl.\n⬥ Can be overcharged to have 5x capacity by Death.\n⬥ Upgradeable at the Mazcab Armoursmith to increase its damage bonus"
+        "value": "*Note: more details available [here](https://runescape.wiki/w/Ring_of_Death)*\n⬥ **Effect**: revived to full HP, 3 min cooldown; activates after Sign of Life <:signoflife:1005552211847295076>\n\u00a0\u00a0\u00a0\u00a0• A ramping DoT is applied which lasts until death or return to a bank area.\n\u00a0\u00a0\u00a0\u00a0\u00a0\u00a0\u00a0\u00a0- Taking off <:RoD:513190159462825984> triples the damage until it is re-equipped.\n⬥ Upgradeable at the Mazcab Armoursmith to increase its damage bonus"
```

Comment:
I would probably keep the 5x capacity section, but mention that it is not recommended as it is no longer worth it.

Reply:
Okay

Reply:
Removed suggested secondary (adren gaining) effect and condensed as suggested by Archeus